WordPress/Classes/ViewRelated/NUX/LoginViewController.swift

```diff
@@ -12,19 +15,27 @@ extension LoginWithLogoAndHelpViewController where Self: UIViewController {
         let imageView = UIImageView(image: image.imageWithTintColor(UIColor.white))
         navigationItem.titleView = imageView
     }
+
     func addHelpButtonToNavController() -> (UIButton, WPNUXHelpBadgeLabel) {
         let helpButtonMarginSpacerWidth = CGFloat(-8)
         let helpBadgeSize = CGSize(width: 12, height: 10)
         let helpButtonContainerFrame = CGRect(x: 0, y: 0, width: 44, height: 44)
 
-        NotificationCenter.default.addObserver(self, selector: #selector(NUXAbstractViewController.handleHelpshiftUnreadCountUpdated(_:)), name: NSNotification.Name.HelpshiftUnreadCountUpdated, object: nil)
+        NotificationCenter.default.addObserver(forName: NSNotification.Name.HelpshiftUnreadCountUpdated, object: nil, queue: nil) { [weak self](notification) in
```

Comment:
You can drop `NSNotification.Name`, since its inferred in the context (just `.HelpshiftUnreadCountUpdated` should work).

Ain't that *AWESOME*? (it's called 'extensible enums', we've recently stumbled upon that). Works as follows:

```
extension String {
    static let someConstant = "Something"
}

func someDummyMethod(string: String) { }
```

Whenever you call `someDummyMethod`, you can just use `.someConstant` as a parameter (Since the type is inferred). `NSNotification.Name` works exactly the same way,